### ISSUE TITLE
use alt attribute on image tags

### ISSUE THREAD

**Issue by rimmesbe** [BlueAdept247/prj-rev-bwfs-dasmoto#4]
Try to get in the habit of describing your images with the alt attribute. This description is used by screen readers, will show up if the image fails to load, and makes your code look more professional.

https://github.com/BlueAdept247/prj-rev-bwfs-dasmoto/blob/master/Art%20Store/index.html#L11
